lib/src/base-styles/_nx-tables.scss

```diff
@@ -119,10 +119,16 @@
   position: relative;
   z-index: 0;
 
-  .nx-cell--header {
+  > thead tr:nth-child(1) th {
     position: sticky;
     top: 0;
-    z-index: 1;
+    z-index: 1
+  }
+
+  > thead tr:nth-child(2) th {
+    position: sticky;
+    top: 45px;
+    z-index: 1
   }
```

Comment:
Some redundancy here could be removed:
```
> thead {
  .nx-cell-header {
    position: sticky;
    z-index: 1;
  }

  > .nx-table-row {
    &:nth-child(1) > .nx-cell--header {
      top: 0;
    }

    &:nth-child(2) > .nx-cell--header {
      top: 45px;
    }
  }
}

Reply:
not longer needed with: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/a954210b17845f9ed60e40fd1bf15de1f5723c9a